test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -6,13 +6,46 @@
 import unittest
 import sys
 import onnxruntime  # noqa
-
+import torch
+import numpy as np
+import io
 
 class TestONNXRuntime(unittest.TestCase):
 
     def test_onnxruntime_installed(self):
         self.assertTrue('onnxruntime' in sys.modules)
 
+    def test_full(self):
+        class FullModel(torch.nn.Module):
+            def forward(self, x):
+                return torch.full((3, 4), x, dtype=torch.long)
+
+        x = torch.tensor(12)
+        model = FullModel()
+        output = model(x)
+
+        f = io.BytesIO()
+        torch.onnx.export(model, x, f)
+
+        ort_sess = onnxruntime.InferenceSession(f.getvalue())
+        ort_outs = ort_sess.run(None, {ort_sess.get_inputs()[0].name: x.numpy()})
+        np.allclose(output.numpy(), ort_outs[0])
+
+        class FullModelScripting(torch.jit.ScriptModule):
```

Comment:
Also you can create this test case into 2 small case, test_full_trace and test_full_script.